sections/futures/TradeCrossMargin/TradeCrossMargin.tsx

```diff
@@ -63,7 +63,7 @@ export default function TradeCrossMargin({ isMobile }: Props) {
 			if (!invalidLabel || !price) {
 				onTradeOrderPriceChange(price);
 			}
-			setOrderPrice(price);
+			setOrderPrice(marketAssetRate.toString(2));
```

Comment:
This would show the current market rate, which we don't want here. The default field output depends on the order type selected and varies.

Make sure to use `setOrderPrice(price)` and have a look how you could add two decimals to the existing output